public/index.html

```diff
@@ -35,19 +35,24 @@ <h1 class="alignright">MapGT</h1>
 
     <div class="right-panel">
         <div>
-            <div id="card" class="card modal show-modal">
+            <!-- <div id="card" class="card modal show-modal">
                 <div class="additional">
-                    <div class="event-card">
-                        <h2>Now</h2>
-                        <p id="current-event"></p>
-                    </div>
-                    <div class="more-info">
-                        <h2>Upcoming</h2>
-                        <div id="upcoming-events" style="white-space: pre-line"></div>
-                    </div>
+                    
                 </div>
                 <div class="general">
                 </div>
+            </div> -->
+            <div class="toppanel">
+                <div class="event-card">
+                    <h2>Now</h2>
+                    <p id="current-event"></p>
+                </div>
+            </div>
+            <div class="bottompanel">
+            <div class="more-info">
+                <h2>Upcoming</h2>
+                <div id="upcoming-events" style="white-space: pre-line"></div>
```

Comment:
We shouldn't have any in-line style – add `white-space: pre-line` to CSS for `.upcoming-events`